frontend/src/Utilities/Number/formatAge.js

```diff
@@ -1,17 +1,19 @@
+import translate from 'Utilities/String/translate';
+
 function formatAge(age, ageHours, ageMinutes) {
   age = Math.round(age);
   ageHours = parseFloat(ageHours);
   ageMinutes = ageMinutes && parseFloat(ageMinutes);
 
   if (age < 2 && ageHours) {
     if (ageHours < 2 && !!ageMinutes) {
-      return `${ageMinutes.toFixed(0)} ${ageHours === 1 ? 'minute' : 'minutes'}`;
+      return `${ageMinutes.toFixed(0)} ${ageHours === 1 ? translate('Minute').toLowerCase() : translate('Minutes').toLowerCase()}`;
```

Comment:
If we don't need `Minute`, `Hour`, `Day` and the plural values capitalized I think we're better off not calling the `toLowerCase` after translating

Reply:
we do have Day capitalized being used elsewhere, so I chose to just capitalize them all and lowercase them here.
Other wise we end up with a mix of `Day`  and `days`

Reply:
hmm, thinking about it more, what if a particular locale doesn't want lowercase for their indication. I will create new keys for this